package.json

```
@@ -2,7 +2,7 @@
   "name": "ledger-live-desktop",
   "productName": "Ledger Live",
   "description": "Ledger Live - Desktop",
-  "version": "2.1.0",
+  "version": "2.0.0",
```

```suggestion
  "version": "2.1.0",
```

(unless it was on purpose to help QA?)

---

not on purpose, I forgot it, I'm getting really slow